src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/AggregationOptionTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.unittest;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static com.amazon.opendistroforelasticsearch.sql.util.SqlExplainUtils.explain;
+
+/**
+ * Unit test class for feature of aggregation options: DISTINCT, ALL, UNIQUE, DEDUPLICATION
+ */
+public class AggregationOptionTest {
```

Comment:
I think test cases in this class is actually IT and should be moved to `AggregationIT`. To do UT, we can create AST by `SqlParserUtils.parse`, pass it to `makeField` and assert return value `Field`.